README.rst

```diff
@@ -162,8 +162,19 @@ Setup
     go_rules_dependencies()
     go_register_toolchains()
 
-  You can add more external dependencies to this file later (see
-  `go_repository`_).
+* You can add more external dependencies to this file later (see
+  `go_repository`_). This rule is in Gazelle and will require you to load it.
+
+  .. code:: bzl
+
+  load("@bazel_gazelle//:deps.bzl", "go_repository")
```

Comment:
Make sure to indent this block of code. Without the indentation, ReST interprets this as part of the main text. The `..code` line also needs to be intended at the same level as the rest of the list item. See other `.. code` blocks for examples.